### PR TITLE
conversion to work without docker/mongo db connect update

### PR DESCRIPTION
**Pull request by TheHuns**
Removed all docker files and added a new package.json file to root of project. Updated Readme with new explanation on how to get up and running.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,9 @@
-*/node_modules/
\ No newline at end of file
+*/node_modules/
+node_modules
+
+package-lock.json
+*/package-lock.json
```
Comment (andreareed) [inline .gitignore:5]: Why ignore the lock files?

File: README.md [hunk 2]
```diff
@@ -1,28 +1,47 @@
...
-To get started, run the following command in the root of the project:
+---
+
+### Set-up
 
+To get started, run one of the following commands in the root of the project:
+
+```
+npm install
+yarn install
```
Comment (andreareed) [inline README.md:16]: We should probably come to a consensus on which we want to use, or the lock files can become out of sync.  Is that why they were added to `.gitignore`?

**Review by andreareed — COMMENTED** [review]
Nice work!  Coupla questions for you

File: README.md (hunk 2 above)
Comment (TheHuns) [reply README.md:16]: Yeah i had left them out so people could choose the package manger they prefer but i see now how that could complicate things

**Pull request closed without merging**